<!DOCTYPE html>
<html><head><title>ClientDiagnostics</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 write to ClientDiagnostics</div><div class="rA">Azure.Monitor.Query (1)</div><div class="rG" id="Azure.Monitor.Query"><div class="rF"><div class="rN">Generated\QueryRestClient.cs (1)</div>
<a href="../Generated/QueryRestClient.cs.html#34"><b>34</b><i>ClientDiagnostics</i> = clientDiagnostics ?? throw new ArgumentNullException(nameof(clientDiagnostics));</a>
</div>
</div>
<div class="rH">6 references to ClientDiagnostics</div><div class="rA">Azure.Monitor.Query (6)</div><div class="rG" id="Azure.Monitor.Query"><div class="rF"><div class="rN">Generated\QueryRestClient.cs (6)</div>
<a href="../Generated/QueryRestClient.cs.html#88"><b>88</b>throw await <i>ClientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/QueryRestClient.cs.html#121"><b>121</b>throw <i>ClientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/QueryRestClient.cs.html#177"><b>177</b>throw await <i>ClientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/QueryRestClient.cs.html#210"><b>210</b>throw <i>ClientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/QueryRestClient.cs.html#254"><b>254</b>throw await <i>ClientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/QueryRestClient.cs.html#281"><b>281</b>throw <i>ClientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
</div>
</div>
</body></html>